apis/virtualKubelet/v1alpha1/namespacemap_types.go

```diff
@@ -20,24 +20,35 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+type mappingPhase string
+
+const (
+	MappingAccepted mappingPhase = "Accepted"
+	MappingRefused  mappingPhase = "Refused"
+)
+
+type RemoteNamespaceStatus struct {
+	// RemoteNamespace is the name chosen by the user at creation time (when he puts labels on his local namespace)
+	RemoteNamespace string `json:"remoteNamespace,omitempty"`
+	// Phase is the remote Namespace's actual status (READY,FAILED)
```

Comment:
align the comment with the actual possible phase values

Reply:
Sure